docs/source/meta/installing.rst

```diff
@@ -2,12 +2,44 @@
 Installing
 ************
 
-Install the package:
+.. note::
+
+    There is no pypi or conda candidate for this package yet.
+    
+
+You can install the development version of the software by following the instructions below.
+
+.. warning::
+    
+    DeltaMetrics is currently in development, with a constantly changing API.
+    We make no guarantee of stability, correctness, or functionality.
+
+
+
+Latest version install
+----------------------
+
+Follow these instructions to install the latest development version of DeltaMetrics.
+
+.. note:: 
+    Developers should follow the developer instructions in the :doc:`../guides/devguide`.
+
+Install by cloning:
 
 .. code:: console
 
-    $ pip install deltametrics
+    $ git clone https://github.com/DeltaRCM/DeltaMetrics.git
 
-.. note::
-	
-	THE PACKAGE IS NOT AVAILABLE ON PYPI YET
+and then ``cd`` into the directory and install editable copy.
+
+.. code:: console
+
+    $ cd deltametrics
```

Comment:
Nit-picking, but I would prefer `cd DeltaMetrics` -- at least that's how the directory is created for me when I clone it.

Reply:
yep, I got it right in the dev guide, but wrong here. Fixed, and will commit.